src/xpk/core/blueprint/blueprint_generator.py

```diff
@@ -354,6 +360,289 @@ def _get_a3_mega_blueprint_dependencies(
+                f"$(concat([{{network={cluster_name}-a3u-net-1.network_name,"
+                f" subnetwork={cluster_name}-a3u-net-1.subnetwork_name,"
+                f' subnetwork_project="{project_id}", nic_type="GVNIC",'
+                " queue_count=null, network_ip=null, stack_type=null,"
+                " access_config=[{nat_ip=null, public_ptr_domain_name=null,"
+                " network_tier=null}], ipv6_access_config=[],"
+                " alias_ip_range=[]}],"
+                f" {cluster_name}-a3u-rdma-net.subnetwork_interfaces_gke))"
+            ),
+        },
+        outputs=["instructions"],
+    )
+    gpu_pool_id = f"{cluster_name}-a3u-ultragpu-pool"
+    gpu_pool = DeploymentModule(
+        id=gpu_pool_id,
+        source="github.com/GoogleCloudPlatform/cluster-toolkit.git//modules/compute/gke-node-pool?ref=e0c690b",
+        use=[cluster_id],
+        settings={
+            "machine_type": "a3-ultragpu-8g",
+            "auto_upgrade": True,
+            "zones": [zone],
+            "disk_type": "hyperdisk-balanced",
+            "disk_size_gb": a3ultra_node_pool_disk_size_gb,
+            "static_node_count": static_node_count,
+            "guest_accelerator": [{
+                "type": "nvidia-h200-141gb",
+                "count": 8,
+                "gpu_driver_installation_config": {
+                    "gpu_driver_version": "LATEST"
+                },
+            }],
+            "additional_networks": (
+                f"$(concat([{{network={cluster_name}-a3u-net-1.network_name,"
+                f" subnetwork={cluster_name}-a3u-net-1.subnetwork_name,"
+                f' subnetwork_project="{project_id}", nic_type="GVNIC",'
+                " queue_count=null, network_ip=null, stack_type=null,"
+                " access_config=[{nat_ip=null, public_ptr_domain_name=null,"
+                " network_tier=null}], ipv6_access_config=[],"
+                " alias_ip_range=[]}],"
+                f" {cluster_name}-a3u-rdma-net.subnetwork_interfaces_gke))"
+            ),
+        },
+        outputs=["instructions"],
+    )
+    if extended_reservation is not None:
+      gpu_pool.settings["reservation_affinity"] = {
+          "consume_reservation_type": "SPECIFIC_RESERVATION",
+          "specific_reservations": [{"name": extended_reservation}],
+      }
+    tas_install_id = "topology-aware-scheduler-install"
+    tas_install = DeploymentModule(
+        id=tas_install_id,
+        source="github.com/GoogleCloudPlatform/cluster-toolkit.git//community/modules/compute/gke-topology-scheduler?ref=e0c690b",
+        use=[cluster_id],
+    )
+    workload_manager_install_id = "workload-manager-install"
+    workload_manager_install = DeploymentModule(
+        id=workload_manager_install_id,
+        source="github.com/GoogleCloudPlatform/cluster-toolkit.git//modules/management/kubectl-apply?ref=e0c690b",
+        use=[cluster_id],
+        settings={
+            "kueue": {"install": True, "version": "v0.9.1"},
```

Comment:
we need kueue config same as a3mega for this. probably with a little modification.